api/src/main/java/marquez/db/migrations/V44_2_BackfillJobsWithParents.java

```diff
@@ -0,0 +1,154 @@
+package marquez.db.migrations;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import marquez.common.Utils;
+import marquez.service.models.LineageEvent.ParentRunFacet;
+import org.flywaydb.core.api.MigrationVersion;
+import org.flywaydb.core.api.migration.Context;
+import org.flywaydb.core.api.migration.JavaMigration;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.result.ResultProducers;
+
+@Slf4j
+public class V44_2_BackfillJobsWithParents implements JavaMigration {
```

Comment:
_Minor_: I'd make it explicit that it's for parent runs `BackfillJobsWithParentRuns`